tools/python/test_server/start_server.py

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env python3
 
+"""
+    Run from `$OMIM_ROOT/tools/python/test_server` dir
+"""
+
 import subprocess
 
 SERVER_RUNNABLE = 'server/testserver.py'
+LOG_FILE = 'server.log'
```

Comment:
Is a temporary file really needed? Why not use a pipe, for example, like this: https://stackoverflow.com/a/67702119 ?

Reply:
Because it doesn't work with our test server implementation. I've already tried everything possible. Only this solution works.